include/oneapi/dpl/pstl/hetero/dpcpp/sycl_iterator.h

```diff
@@ -18,6 +18,8 @@
 
 #include <CL/sycl.hpp>
 
+#define __SYCL_2020_NOINIT_MACRO (__LIBSYCL_MAJOR_VERSION >= 5 && __LIBSYCL_MINOR_VERSION >= 2)
```

Comment:
Could you please point where you found that actual values for macros are 5 and 2?

Reply:
Got information here: https://intel.github.io/llvm-docs/doxygen/version_8hpp.html